starter-core/src/main/java/io/micronaut/starter/feature/opentelemetry/OpenTelemetryHttp.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.opentelemetry;
+
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.dependencies.Dependency;
+import io.micronaut.starter.feature.other.Management;
+import io.micronaut.starter.feature.server.MicronautServerDependent;
+import io.micronaut.starter.feature.tracing.TracingFeature;
+import jakarta.inject.Singleton;
+
+@Singleton
+public class OpenTelemetryHttp implements TracingFeature, MicronautServerDependent {
+
+    private static final Dependency.Builder MICRONAUT_OPEN_TELEMETRY_HTTP = Dependency.builder()
+            .groupId("io.micronaut.tracing")
+            .artifactId("micronaut-tracing-opentelemetry-http")
+            .compile();
+
+    @NonNull
+    @Override
+    public String getName() {
+        return "tracing-opentelemetry-http";
+    }
+
+    @NonNull
+    @Override
+    public String getTitle() {
+        return "OpenTelemetry HTTP";
+    }
+
+    @Override
+    @NonNull
+    public String getDescription() {
+        return "Enables the creation of span objects HTTP server request, client request, server response and client response";
+    }
+
+    @Override
+    public void apply(GeneratorContext generatorContext) {
```

Comment:
This brings in "io.micronaut.tracing:micronaut-tracing-opentelemetry", but we might want to add it as a dependency that way they can remove the http dependency if they don't want to trace the "http" calls.

Reply:
I did this commit https://github.com/micronaut-projects/micronaut-starter/pull/1295/commits/d402e7e65b21124e162f1a166beb002c30d508ba

Does it solve this issue?